projects/core/utils/mask/create-number-mask.ts

```diff
@@ -16,6 +16,7 @@ const NON_ZERO_DIGIT = /[1-9]/;
 export function tuiCreateNumberMask({
     allowDecimal = false,
     decimalSymbol = ',',
+    thousandSymbol = CHAR_NO_BREAK_SPACE,
```

Comment:
I wonder if this can be called a breaking change 🤔 @MarsiBarsi @vladimirpotekhin @nsbarsukov what do you think?

Reply:
As I can see, there are no breaking changes

Reply:
I just mean it is an exported function and the order of arguments have changed. Some people might have used it in their projects. Maybe it would be better to move to the last argument or change `decimalSymbol` to `format` with `string | object` type and an an overload, deprecating current interface.

Reply:
This is an object, the order doesn't matter

Reply:
Oh my bad, didn't notice the brackets )